src/hooks/useMediaVisible.ts

```diff
@@ -19,14 +19,15 @@ const PRIVATE_JOIN_RULES: JoinRule[] = [JoinRule.Invite, JoinRule.Knock, JoinRul
 
 /**
  * Should the media event be visible in the client, or hidden.
- * @param eventId The eventId of the media event.
```

Comment:
Can we improve docs rather than make them worse? Notably would be good to document the behaviour if `mxEvent === undefined`